### PR TITLE
Refactor site information settings, refs #13028

### PR DESCRIPTION
**Pull request by mcantelon**
Refactored the site information settings page to use the settings editing
parent class.

### CODE REVIEW

File: apps/qubit/modules/settings/actions/siteInformationAction.class.php [hunk 1]
```diff
@@ -27,103 +27,59 @@
...
-    // Save Site Description
-    $siteDescription = $thisForm->getValue('site_description');
-    $siteDescSetting = QubitSetting::getByName('siteDescription');
+        break;
 
+      case 'siteDescription':
+        $this->form->setWidget($name, new sfWidgetFormInput);
+        $this->form->setValidator($name, new sfValidatorString(array('required' => false)));
+        $labelText = $this->i18n->__('Site description');
+        $this->form->getWidgetSchema()->$name->setLabel($labelText);
```
Comment (djjuhasz) [inline apps/qubit/modules/settings/actions/siteInformationAction.class.php]: Nitpick: The `$labelText` declaration isn't strictly necessary as it is only used the one time.

File: apps/qubit/modules/settings/templates/_i18n_form_field.php [hunk 2]
```diff
@@ -1,5 +1,11 @@
-<?php echo $form->$name
-  ->renderLabel($label) ?>
+<?php if (!empty($label)): ?>
+  <?php echo $form->$name
```
Comment (djjuhasz) [inline apps/qubit/modules/settings/templates/_i18n_form_field.php]: Nitpick: breaking this over two lines seems unnecessary.

File: apps/qubit/modules/settings/templates/_i18n_form_field.php [hunk 3]
```diff
@@ -1,5 +1,11 @@
...
+    ->renderLabel($label) ?>
```
Comment (djjuhasz) [inline apps/qubit/modules/settings/templates/_i18n_form_field.php:3]: Nitpick: breaking this over two lines seems unnecessary.

File: apps/qubit/modules/settings/templates/siteInformationSuccess.php [hunk 4]
```diff
@@ -27,39 +27,42 @@
...
-                <div class="default-translation"><?php echo $sourceCultureHelper ?></div>
-              <?php endif; ?>
-              <?php echo $siteInformationForm['site_title']->render() ?>
+              <?php echo $form->siteTitle->renderLabel('Site title', array('title' => 'The name of the website for display in the header')) ?>
+            </td>
+            <td>
+              <?php echo get_partial('settings/i18n_form_field',
+                array(
+                  'name' => 'siteTitle',
+                  'label' => false,
```
Comment (djjuhasz) [inline apps/qubit/modules/settings/templates/siteInformationSuccess.php]: Nitpick: I think `'label' => null,` is more appropriate in this context.

File: apps/qubit/modules/settings/templates/siteInformationSuccess.php [hunk 5]
```diff
@@ -27,39 +27,42 @@
...
-            <td><?php echo $siteInformationForm['site_description']->renderLabel(null,
-              array('title' => __('A brief site description or &quot;tagline&quot; for the header'))) ?></td>
             <td>
-              <?php if (strlen($error = $siteInformationForm['site_description']->renderError())): ?>
-                <?php echo $error ?>
-              <?php elseif ($sourceCultureHelper = $siteDescription->getSourceCultureHelper($culture)): ?>
-                <div class="default-translation"><?php echo $sourceCultureHelper ?></div>
-              <?php endif; ?>
-              <?php echo $siteInformationForm['site_description']->render() ?>
+              <?php echo $form->siteDescription->renderLabel('Site description', array('title' => 'A brief site description or &quot;tagline&quot; for the header')) ?>
```
Comment (djjuhasz) [inline apps/qubit/modules/settings/templates/siteInformationSuccess.php]: Nitpick: long line

**Review by djjuhasz — COMMENTED** [review]
Looks great Mike! I just suggested a few nitpicky style improvements. 👍

**Comment by mcantelon** [comment]
Thanks @djjuhasz! Good feedback... I've updated the PR with fixes.

**Pull request merged**